README.md

```diff
@@ -115,7 +115,10 @@ func main() {
     // Create the AlphaZero structure 
     a := agogo.New(g, conf)
     // Launch the learning process
-    a.Learn(5, 30, 200, 30) // 5 epochs, 50 episode, 100 NN iters, 100 games.
+    err := a.Learn(5, 50, 100, 100) // 5 epochs, 50 episode, 100 NN iters, 100 games.
+    if err != nil {
+        log.Println(err)
+    }
```

Comment:
Without this, a.Learn fails silently because of #3 
I've also hit #4 (which I see has been fixed), but a `go get -u github.com/gorgonia/agogo` doesn't move things forward because I suspect the 0.1.2 tag doesn't exist.